backend/app/services/extract_draft_enrollments_to_remind.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class ExtractDraftEnrollmentsToRemind
+  attr_reader :enrollment
+
+  def call
+    if created_at_updated_at_last_events?
+      created_at_updated_at_last_events_ids =
+        draft_enrollments_with_last_update_or_create_events.pluck(:enrollment_id).flatten
```

Comment:
draft_enrollments_with_last_update_or_create_events is called twice, we must improve this.

maybe you can call `draft_enrollments_with_last_update_or_create_events` first then test that it's not an empty array.